docs/code-quality/c6284.md

```diff
@@ -14,44 +14,45 @@ ms.workload:
   - "multiple"
 ---
 # C6284
-warning C6284: object passed as parameter '%d' when string is required in call to \<function>.
+warning C6284: object passed as parameter when string is required in call to \<function
```

Comment:
is \\< a typo because you are missing the closing >. what does this do in markdown.

Reply:
I didn't mean to remove the closing >, adding it back on.  It did not affect the markdown.